examples/minio/00-minio-deployment.yaml

```diff
@@ -63,7 +63,7 @@ metadata:
   labels:
     component: minio
 spec:
-  type: ClusterIP
+  type: NodePort
```

Comment:
@skriss do you want to make this change, or instead possibly consider #1006 combined with some form of ingress for minio?

Reply:
I feel like the `NodePort` path is the lowest-friction one for a quickstart, while #1006 + ingress is probably more of a production-like setup...so I guess both imho?

Reply:
I think having both documented would be ideal. Although I would recommend ClusterIP+ingress as one option, and NodePort as the other. In other words, don't make a blanked change from ClusterIP to NodePort.

Reply:
Fine with me to document both as alternatives. We might want to have 2 alternate sets of YAML in that case.

Reply:
@Bradamant3 have you seen andy's comment on this re: documenting both a nodeport and an ingress approach here?